src/pages/index.js

```diff
@@ -64,28 +65,44 @@ function createCard(items) {
     function (card) {
       confirmPopup.openPopup();
       confirmPopup.handleConfirm(function () {
+        confirmPopup.handleButtonLoading(true);
         const cardId = items._id;
-        newApi.deleteApiCard(cardId).then(card);
-        card.deleteCard();
+        newApi
+          .deleteApiCard(cardId)
+          .then(() => {
+            card.deleteCard();
+            confirmPopup.closePopup();
+            confirmPopup.handleButtonLoading(false);
+          })
+          .catch((error) => {
+            console.error("An error occured:", error);
+            return Promise.reject(error);
```

Comment:
`return Promise.reject(error);` shouldn't be used in `catch` because it throws an error again but `catch` should keep it inside and you need to handle it

**Please, correct it everywhere**